src/app/api/routes/installations.py

```diff
@@ -53,8 +53,12 @@ async def delete_installation(installation_id: int = Path(..., gt=0)):
     return await crud.delete_entry(installations, installation_id)
 
 
+<<<<<<< HEAD
 @router.post("/site-devices/{site_id}", response_model=List[int],
              summary="Get all devices related to a specific site")
+=======
+@router.get("/site-devices/{site_id}", response_model=List[int])
+>>>>>>> master
```

Comment:
From master, you only need to keep the post -> get edit :+1:

Reply:
oops, I did not save my work so my previous commit was useless.  I think we are good now